libs/SalesforceSDK/src/com/salesforce/androidsdk/rest/RestRequest.java

```diff
@@ -370,6 +373,21 @@ public static RestRequest getRequestForUserInfo() {
 		return new RestRequest(RestMethod.GET, RestEndpoint.LOGIN, RestAction.USERINFO.getPath(), (RequestBody) null, null);
 	}
 
+	/**
+	 * Request to generate URL to bridge into UI sessions (a front door URL)
+	 * Applications should use that API instead of building front door URLs directly
+	 * @param redirectUri A relative path that points to where the user is redirected when their new session begins.
+	 * @return RestRequest object that requests single access URL.
+	 * @see <a href="https://help.salesforce.com/s/articleView?id=sf.frontdoor_singleaccess.htm">https://help.salesforce.com/s/articleView?id=sf.frontdoor_singleaccess.htm</a></a>
```

Comment:
Trivial, but I suspect there's an extra anchor closing tag here and perhaps the content of the link could be more descriptive.